libr/core/rvc.c

```diff
@@ -1003,19 +937,7 @@ R_API bool r_vc_new(const char *path) {
 }
 
 R_API bool r_vc_checkout(const char *rp, const char *bname) {
-	switch (repo_exists (rp)) {
-	case 1:
-		break;
-	case 0:
-		eprintf ("No repo in %s\nCan't checkout\n", rp);
-		return false;
-	case -1:
-		eprintf ("Can't checkout\n");
-		return false;
-	case -2:
-		eprintf ("Can't checkout");
-		return false;
-	}
+	CHECK_REPO (rp, "Can't checkout");
```

Comment:
if (!check_repo (rp, true)) {return false;}

Reply:
I think I may have misnamed repo_exists the purpose of it is not to check wheter or not a repo exists but to run digonstics on it. Do you think I should keep the functionality or change it?